experimental/GModule/src/Cohomology.jl

```diff
@@ -136,14 +146,14 @@ group `H`, return the `ZZ[H]` module.
 
 Note: we do not check that this defined indeed a `ZZ[H]` module.
 """
-function gmodule(H::Oscar.GAPGroup, ac::Vector{<:Map})
+function gmodule(H::Union{Nothing, Oscar.GAPGroup}, ac::Vector{<:Map})
```

Comment:
Why is this is necessary? What is supposed to happen when `H` is equal to `nothing`.